src/nvim/window.c

```diff
@@ -6000,7 +5982,7 @@ void win_drag_status_line(win_T *dragwin, int offset)
   cmdline_row = row;
   p_ch = MAX(Rows - cmdline_row, ui_has(kUIMessages) ? 0 : 1);
   curtab->tp_ch_used = p_ch;
-  redraw_all_later(SOME_VALID);
+  redraw_all_later(NOT_VALID);
```

Comment:
is this necessary? tests still pass with `SOME_VALID` .

Reply:
> is this necessary? tests still pass with `SOME_VALID` .

I think I initially made the change to fix a certain bug but it turned out to not be the proper fix but I forgot to reverse this after I actually made the proper fix, I'll switch it back to `SOME_VALID`